docs/4-user-guide/6-vscode.md

```diff
@@ -4,20 +4,22 @@ Zarf uses its own [schema](https://github.com/defenseunicorns/zarf/blob/main/zar
 
 ## Adding schema validation
 
-1. Open VS Code's `settings.json` file with `CTRL/CMD + SHIFT + P` and search for `Preferences: Open User Settings (JSON)`.
-2. Add the below to your config, or modify the existing `yaml.schemas` object to include the Zarf schema.
-
-:::note
-
-The [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) by RedHat is a prerequisite for this feature.
-
-:::
+1. Open VS Code
+2. Install the [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) by RedHat.
+3. Open the VS Code`settings.json` file by typing `CTRL/CMD + SHIFT + P`. 
+4. Use the search bacr to search for `Preferences: Open User Settings (JSON)`.
```

Comment:
Technically CMD + Shift + P opens the command palette and then searching for `Preferences: Open User Settings (JSON)` is what opens `settings.json`

Reply:
yeah lol - fair point.